.github/workflows/ubi8-google-api-python-client-pr.yaml

```diff
@@ -0,0 +1,37 @@
+name: ubi8-google-api-python-client-pr
+on:
+  pull_request:
+    paths:
+      - utilities/ubi8-google-api-python-client/**
+jobs:
+  build:
+    env:
+      context: utilities/ubi8-google-api-python-client
+      image_name: ubi8-google-api-python-client
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - name: Check and verify version.json
+        id: check_version
+        run: |
+          echo -n ::set-output name=IMAGE_TAGS::
+          echo $(jq -r '.version' ${context}/version.json)
+      - name: Build image
+        uses: docker/build-push-action@v1
+        with:
+          path: ${{ env.context }}
+          push: false
+          repository: ${{ env.image_name }}
+          tags: ${{ steps.check_version.outputs.IMAGE_TAGS }}
+      - name: Test image
+        run: |
+          echo "Running: docker run ${image_name}:${{ steps.check_version.outputs.IMAGE_TAGS }} git --version"
+          docker run ${image_name}:${{ steps.check_version.outputs.IMAGE_TAGS }} git --version
+          echo "Running: docker run ${image_name}:${{ steps.check_version.outputs.IMAGE_TAGS }} pip3 --version"
+          docker run ${image_name}:${{ steps.check_version.outputs.IMAGE_TAGS }} pip3 --version
+          echo "Running: docker run ${image_name}:${{ steps.check_version.outputs.IMAGE_TAGS }} pip3 show google-api-python-client"
+          docker run ${image_name}:${{ steps.check_version.outputs.IMAGE_TAGS }} pip3 show google-api-python-client
+          echo "Running: docker run ${image_name}:${{ steps.check_version.outputs.IMAGE_TAGS }} pip3 show google-auth-httplib2"
+          docker run ${image_name}:${{ steps.check_version.outputs.IMAGE_TAGS }} pip3 show google-auth-httplib2
+          echo "Running: docker run ${image_name}:${{ steps.check_version.outputs.IMAGE_TAGS }} pip3 show google-auth-oauthlib"
+          docker run ${image_name}:${{ steps.check_version.outputs.IMAGE_TAGS }} pip3 show google-auth-oauthlib
```

Comment:
Could these be combined into one test? Not a big deal if not, just seems nice to only do one `docker run`.
`pip3 --version && pip3 show google-api-python-client google-auth-httplib2 google-auth-oauthlib`